xbmc/cores/AudioEngine/Sinks/pipewire/Pipewire.cpp

```diff
@@ -68,10 +78,24 @@ bool CPipewire::Start()
 
   if (ret == -ETIMEDOUT)
   {
-    CLog::Log(LOGDEBUG, "CAESinkPipewire::{} - timed out out waiting for synchronization",
-              __FUNCTION__);
+    CLog::Log(LOGDEBUG, "Pipewire: timed out out waiting for synchronization");
     return false;
   }
 
   return true;
 }
+
+std::unique_ptr<CPipewire> CPipewire::Create()
+{
+  struct PipewireMaker : public CPipewire
+  {
+    using CPipewire::CPipewire;
```

Comment:
This must allow for adding functionality to PipewireMaker in the future?

Reply:
It's purely for allowing the use of a private constructor with `std::make_unique`